### PR TITLE
Added class and fullscreen attributes for the Vimeo helper

### PR DESCRIPTION
**Pull request by themao**
As seen in other providers, providing class attribute for a twig helper (and also some Vimeo iframe attributes)

### CODE REVIEW

File: Provider/VimeoProvider.php [hunk 1]
```diff
@@ -61,11 +61,13 @@ public function getHelperProperties(MediaInterface $media, $format, $options = a
...
-            'id'          => $player_parameters['js_swf_id'],
-            'frameborder' => isset($options['frameborder']) ? $options['frameborder'] : 0,
-            'width'       => $box->getWidth(),
-            'height'      => $box->getHeight(),
+            'src'               => http_build_query($player_parameters),
+            'id'                => $player_parameters['js_swf_id'],
+            'frameborder'       => isset($options['frameborder']) ? $options['frameborder'] : 0,
+            'width'             => $box->getWidth(),
+            'height'            => $box->getHeight(),
+            'class'             => isset($options['class']) ? $options['class'] : false,
```
Comment (OskarStark) [inline Provider/VimeoProvider.php]: would use an empty string here as default and then check for emptiness in the twig template.
I dont like mixing types

@core23 what do you think?

Comment (core23) [inline Provider/VimeoProvider.php]: :+1: empty string as default value

**Comment by core23** [comment]
Can you please rebase @themao

**Comment by themao** [comment]
@core23 sorry for reacting so long, I've rebased and updated my PR

**Comment by core23** [comment]
You didn't rebase, you've merged everything into your branch @themao

**Comment by themao** [comment]
@core23 okay, now I finally got how to handle the rebase :) Now it's rebased

**Comment by core23** [comment]
Thank you @themao

**Pull request merged**